src/custom/components/AccountDetails/SimpleAccountDetails.tsx

```diff
@@ -0,0 +1,50 @@
+import { Fragment } from 'react'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import { useMultipleActivityDescriptors, groupActivitiesByDay } from 'hooks/useRecentActivity'
+import { supportedChainId } from 'utils/supportedChainId'
+import { LowerSectionSimple, Wrapper, NoActivityMessage } from './styled'
+import { renderActivities } from './AccountDetailsMod'
+import { AccountDetailsProps } from '.'
+import styled from 'styled-components/macro'
+
+type StyledWrapperProps = { $margin?: string }
+type SimpleAccountDetailsProps = Pick<AccountDetailsProps, 'pendingTransactions' | 'confirmedTransactions'> &
+  StyledWrapperProps
+
+const SimpleWrapper = styled(Wrapper)<StyledWrapperProps>`
+  ${({ $margin }) => $margin && `margin: ${$margin};`}
```

Comment:
What happens when `$margin` is falsy?
Is the resulting falsy value included in the output css?

Reply:
falsy in styleds returns nothing, like null or undefined or ''